components/header.js

```diff
@@ -1,30 +1,54 @@
 import Link from 'next/link'
 import Headroom from 'react-headroom'
+import cn from 'classnames';
+import React, {useState} from 'react';
 
 import Container from '../components/container';
 import Navbar from '../components/navbar';
 import PreviewMode from '../components/preview-mode';
 
 export default function Header({ preview, links }) {
+
+  const [isOpen, setIsOpen] = useState(false)
+
+  const toggleMenu = () => {
+    setIsOpen(!isOpen)
+  };
+
   return (
     <Headroom
       disableInlineStyles
-      className="Header"
-    >
+      className="Header">
+
       {preview && <PreviewMode />}
-      <div className="Header-wrapper">
-        <Container className="u-flex u-flexAlignItemsCenter u-flexJustifyBetween">
-          <Link href="/">
-            <a className="Header-brand" aria-label="Link to homepage">
-              <span className="Header-brand-image">
-                <img src="https://via.placeholder.com/150x83.png?text=Logo" alt="Southern Green Hydrogen" className="u-block" />
-              </span>
-            </a>
-          </Link>
-
-          <Navbar links={links} />
+      <div className={cn("Header-wrapper", {'is-open': isOpen})}>
+        <Container className="u-flexCol u-lg-flexRow Header-wrapper-container">
+          <div className="u-flex u-flexAlignItemsCenter u-flexJustifyBetween u-width1of1 u-lg-width1of3">
+            <div className="u-flex u-flexAlignItemsCenter">
+              <Link href="/">
+                <a className="Header-brand-image" aria-label="Link to homepage">
+                </a>
+              </Link>
+              <p className="Header-brand-tagline">Advancing Aotearoa's energy transition through clean fuel</p>
+            </div>
+
+            <div className="Hamburger" onClick={() => toggleMenu()}>
+              <span></span>
+              <span></span>
+            </div>
+          </div>
+
+          <div className="Header-wrapper-right u-flexAlignItemsCenter">
+            <Navbar links={links} />
+            <p className="Header-brand-tagline mobile">Advancing Aotearoa's energy transition through clean fuel</p>
```

Comment:
Use utility classes to hide and show this instead of a `mobile` class. 

e.g. `u-sm-block u-md-hidden`

or if you need to style it differently then use a modifier like this:

e.g. `Header-brand-tagline Header-brand-tagline--small`


```suggestion
            <p className="Header-brand-tagline u-sm-block u-md-hidden">Advancing Aotearoa's energy transition through clean fuel</p>
```